Tests/KlaviyoSwiftTests/StateManagementEdgeCaseTests.swift

```diff
@@ -369,15 +369,30 @@ class StateManagementEdgeCaseTests: XCTestCase {
     // MARK: - set enqueue event uninitialized
 
     @MainActor
-    func testEnqueueEventUninitialized() async throws {
+    func testOpenedPushEventUninitializedAddsToPendingRequests() async throws {
```

Comment:
Tests opened push event is added to pending requests when SDK isn't initialized.